src/clib/solve_rate_cool_g-cpp.cpp

```diff
@@ -837,10 +836,10 @@ int solve_rate_cool_g(
           // Compute dedot and HIdot, the rates of change of de and HI
           //   (should add itmask to this call)
 
-          f_wrap::rate_timestep_g(
-            spsolvbuf.dedot, spsolvbuf.HIdot, anydust, idx_range,
-            spsolvbuf.h2dust, rhoH.data(), itmask.data(), edot.data(),
-            chunit, dom, my_chemistry, my_fields, *my_uvb_rates,
+          grackle::impl::rate_timestep_g(
+            spsolvbuf.dedot, spsolvbuf.HIdot, anydust, spsolvbuf.h2dust,
+            rhoH.data(), itmask.data(), edot.data(),
+            chunit, dom, my_chemistry, my_fields, *my_uvb_rates, idx_range,
```

Comment:
`photo_rate_storage` suggestion 3/4
```suggestion
            chunit, dom, my_chemistry, my_fields, idx_range,
```